nubis/puppet/fluentd.pp

```diff
@@ -4,17 +4,25 @@
 }
 
 # Make fluentd run as root so it can read all log files
-file { "/etc/sysconfig":
-  ensure => "directory",
-  owner  => "root",
-  group  => "root",
-  mode    => '0755',
-}->
-file { "/etc/sysconfig/td-agent":
-  ensure => "present",
-  owner  => "root",
-  group  => "root",
-  source => "puppet:///nubis/files/fluentd.sysconfig",
+if $osfamily == 'RedHat' {
+  file { "/etc/sysconfig":
```

Comment:
Technically not really needed since we assume that it gets created anyway (RedHat should always have this) but it doesn't hurt it anyway